api/config/v1alpha1/validation/validation_test.go

```diff
@@ -355,12 +356,18 @@ func TestValidateCompactionControllerConfiguration(t *testing.T) {
 			matcher:        ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{"Type": Equal(field.ErrorTypeInvalid), "Field": Equal("controllers.compaction.concurrentSyncs")}))),
 		},
 		{
-			name:           "should forbid event threshold less than zero",
-			enabled:        true,
-			eventThreshold: ptr.To(int64(-1)),
-			expectedErrors: 1,
-
-			matcher: ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{"Type": Equal(field.ErrorTypeInvalid), "Field": Equal("controllers.compaction.eventsThreshold")}))),
+			name:            "should forbid event threshold less than zero",
+			enabled:         true,
+			eventsThreshold: ptr.To(int64(-1)),
+			expectedErrors:  1,
+			matcher:         ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{"Type": Equal(field.ErrorTypeInvalid), "Field": Equal("controllers.compaction.eventsThreshold")}))),
+		},
+		{
```

Comment:
The validation returns an error when `TriggerFullSnapshotThreshold` is <= 0 but there is no test if it is zero. Can you add that test as well?